app/scripts/controllers/overview.js

```diff
@@ -1262,17 +1264,17 @@ function OverviewController($scope,
       Logger.log("deploymentconfigs (subscribe)", overview.deploymentConfigs);
     }));
 
-    watches.push(DataService.watch(replicaSetsVersion, context, function(replicaSetData) {
-      overview.replicaSets = replicaSetData.by('metadata.name');
-      groupReplicaSets();
-      updateServicesForObjects(overview.vanillaReplicaSets);
-      updateServicesForObjects(overview.monopods);
-      updatePodWarnings(overview.vanillaReplicaSets);
-      updateLabelSuggestions(overview.vanillaReplicaSets);
-      groupBindings();
-      updateFilter();
-      Logger.log("replicasets (subscribe)", overview.replicaSets);
-    }));
+    // watches.push(DataService.watch(replicaSetsVersion, context, function(replicaSetData) {
```

Comment:
remote commented code or add TODO statement with issue# or task#

Reply:
If this is rebased on master we shouldn't need to comment it out any more also